tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/DynamoDb/IAmazonDynamoDbRequestWithKnownKeys.cs

```diff
@@ -0,0 +1,70 @@
+// <copyright file="IAmazonDynamoDbRequestWithKnownKeys.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.IO;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.DynamoDb
```

Comment:
nitpicky, but it would be beneficial to have a perma link (to the GitHub location) to each of these underlying definitions for future reference